helm/ph-ee-test/values.yaml

```diff
@@ -216,15 +193,34 @@ ph-ee-engine:
 
   channel:
     enabled: true
-    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel-test"
-    imageTag: test
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel"
+    imageTag: v1.5.1
     SPRING_PROFILES_ACTIVE: "bb"
     hostname: "channel2.sandbox.fynarfin.io"
     stub_hostname: "channel-gsma2.sandbox.fynarfin.io"
     DFSPIDS: "rhino,gorilla"
+    redis:
+      host: "127.0.0.1"
```

Comment:
Redis host shouldn't be localhost.

Reply:
@avikganguly01 Remoed redis related config from here as this was added while debugging an error. @somanath21 has added the fix for the error.